diff --git a/sw/linux-pfeng/pfeng-drv.c b/sw/linux-pfeng/pfeng-drv.c
index 81edc1c..f3199e5 100644
--- a/sw/linux-pfeng/pfeng-drv.c
+++ b/sw/linux-pfeng/pfeng-drv.c
@@ -57,8 +57,8 @@
 /* PFE SYS CLK is 300MHz */
 #define PFE_CLK_SYS_RATE			300000000
 
-/* PFE TS CLK is 200MHz */
-#define PFE_CLK_TS_RATE				200000000
+/* PFE TS CLK is 100MHz */
+#define PFE_CLK_TS_RATE				100000000
 
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Jan Petrous <jan.petrous@nxp.com>");
@@ -736,6 +736,10 @@ static int pfeng_drv_probe(struct platform_device *pdev)
 	/* Prepare PTP clock */
 	priv->clk_ptp_reference = 0U;
 	priv->clk_ptp = clk_get(dev, "pfe_ts");
+
+	priv->clk_ptp_reference = PFE_CLK_TS_RATE;
+	HM_MSG_DEV_INFO(dev,"no set clock rate, clk_ptp_reference: %lld\n", priv->clk_ptp_reference);
+	/**********Modified by AC.SYS Milon.Yang*********
 	if (IS_ERR(priv->clk_ptp)) {
 		HM_MSG_DEV_WARN(dev, "Failed to get pfe_ts clock. PTP will be disabled.\n");
 		priv->clk_ptp = NULL;
@@ -753,6 +757,7 @@ static int pfeng_drv_probe(struct platform_device *pdev)
 				priv->clk_ptp_reference = clk_get_rate(priv->clk_ptp);
 		}
 	}
+	***************End of Modify*****************/
 
 	/* Create MDIO buses */
 	pfeng_mdio_register(priv);
